hello_distribution/.tool-versions

```diff
@@ -0,0 +1,2 @@
+erlang 24.0.3
+elixir 1.12.0-otp-24
```

Comment:
We don't have `.tool-versions` files in any of the other examples. It seems like a good idea, but probably better for the base directory and in a different PR.